composer.json

```diff
@@ -98,5 +99,8 @@
         "psr-4": {
             "Piwik\\Tests\\": "tests/PHPUnit/"
         }
+    },
+    "scripts": {
+        "phpstan": "phpstan analyse -c phpstan.neon"
```

Comment:
No need to prefix with `vendor/bin` since composer adds the current path on top of the path stack when running composer scripts.

Reply:
I've added the scripts sections since it's easier to just do `composer run phpstan` than having to remember the full command with the config file (that can be omitted and it will find it) but composer also then hints at what commands are available for the given project.
If you are strongly against that we can revert that.